docs/howto/index.md

```diff
@@ -13,4 +13,5 @@ Back up and restore Ubuntu WSL instances <backup-and-restore>
 Uninstalling UP4W, Ubuntu WSL apps and WSL <uninstalling>
 Configure the Landscape client with UP4W  <set-up-landscape-client>
 Set up a Landscape server within WSL <set-up-landscape-server-in-wsl>
+Create WSL instances on Multiple Windows machines via Landscape API <custom-rootfs-multiple-targets>
```

Comment:
no capital for "Multiple".

also, I would try to avoid _via_ (latinism) where possible and would replace it here with something like "with the".